app/components/ruleRunner.js

```diff
@@ -0,0 +1,9 @@
+export const ruleRunner = (fieldName, fieldLabel, ...validation) => {
+  return;
```

Comment:
`...validations` is an array of functions. loop through and call each validator to get the errorMessages.